src/app/body/create-new-organization/create-new-organization.component.ts

```diff
@@ -64,6 +66,13 @@ export class CreateNewOrganizationComponent implements OnInit {
     this.orgAdminUid = this.authService.getLoggedInUser()
     this.navbarHandler.resetNavbar()
     this.navbarHandler.addToNavbar(this.componentName);
+    ((this.authService.myOrgCollectionsData.forEach(item=>{
+      (item.forEach(item=>{
+        this.OrgDomains.push(item.OrgDomain);
+      }));
+    })))
+
+    
```

Comment:
This part is completely wrong.
This checks the org domains of the organizations in which the user is part of instead you need to check it in all the organzations in the application.

Reply:
resolved